docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/xcluster-replication-setup.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Set up xCluster replication for a universe
+headerTitle: Set up xCluster replication
+linkTitle: Setup
+description: Setting up xCluster replication for a universe
+headContent: Start replication from your source to your target
+menu:
+  preview_yugabyte-platform:
+    parent: xcluster-replication
+    identifier: xcluster-replication-setup
+    weight: 10
+type: docs
+---
+
+## Prerequisites
+
+Create two universes, the source universe which will serve reads and writes, and the target.
+
+Ensure the universes have the following characteristics:
+
+- Both universes are running the same version of YugabyteDB (v2.18.0.0 or later).
+- Both universes have the same [encryption in transit](../../../security/enable-encryption-in-transit/) settings. Encryption in transit is recommended, and you should create the source and target universes with TLS enabled.
+
+    If you [rotate the CA certificate](../../../security/enable-encryption-in-transit/rotate-certificates/) on the source universe, you need to restart the replication so that the target nodes get the new root certificate for TLS verifications.
+
+- They can be backed up and restored using the same [storage configuration](../../../back-up-restore-universes/configure-backup-storage/).
+
+    The storage configuration for copying data from source to target must be accessible from both universes. For example, if AWS S3 is used as the target of backups, the same S3 bucket must be accessible from the source and target universe.
+
+    If an NFS file server is used, the file server or a mirror of the file server in the target region must be accessible at the same mount point on both universes. If there is significant delay in file transfer between the file server and its mirror in a different region, set the [runtime configuration](../../../administer-yugabyte-platform/manage-runtime-config/) `yb.xcluster.sleep_time_before_restore` to the maximum delay in mirroring files across the regions.
+
+- They have enough disk space to support storage of write-ahead logs (WALs) in case of a network partition or a temporary outage of the target universe. During these cases, WALs will continue to write until replication is restored. Consider sizing your disk according to your ability to respond and recover from network or other infrastructure outages.
+
+    In addition, during xCluster replication setup, the source universe retains WAL logs and increases in size. The setup (including copying data from source to target) is not aborted if the source universe runs out of space. Instead, a warning is displayed notifying that the source universe has less than 100 GB of space remaining. Ensure that there is enough space available on the source universe before attempting to set up the replication. A recommended approach would be to estimate that the available disk space on the source universe is the same as the used disk space.
```

Comment:
`A recommended approach would be to estimate that the available disk space on the source universe is the same as the used disk space.`

This doesn't give a good estimate in general. We can remove this line and add a better method of estimating this in the future. (cc @shahrooz1997 )